ios general/ios general/Networking/RequestManager.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+import Alamofire
+
+class RequestManager: NSObject
+{
+    var orgsNames = [String]()
+    
+    static func fetchOrganizations(data: @escaping ([String : AnyObject]) -> Void)
+    {
+        NSLog("Fetching!")
+        Alamofire.request("http://gitlab.faifly.com/ios-general/ios-tutorial/uploads/ffffec34fa4a727922bb1e09cc24b21d/tutorial.json").responseJSON { response in
+            print(response.result)
+            if let JSON = response.result.value
+            {
+                let dictionary = JSON as? [String: AnyObject]
```

Comment:
You're using an optional cast here, and you're force unwrapping the value in the next line.